src/xrGame/ui/UIGameTutorial.h

```diff
@@ -26,6 +26,7 @@ class CUISequencer : public pureFrame, public pureRender, public IInputReceiver
 
 public:
     IInputReceiver* m_pStoredInputReceiver;
+    virtual ~CUISequencer() = default;
     CUISequencer();
```

Comment:
For the consistency, it's better to place the destructor after the constructor.
```suggestion

    CUISequencer();
    virtual ~CUISequencer() = default;

```